src/main/java/org/jenkinsci/test/acceptance/docker/Docker.java

```diff
@@ -156,6 +156,16 @@ private void copyDockerfileDirectoryFromClasspath(Class<? extends DockerContaine
             // Dockerfile is not packaged into a jar file, so copy locally
             copyDockerfileDirectoryFromLocal(dockerfileLocation, dir);
         }
+        // if the fixture is dynamic (needs to know something about our environment then process it.
+        if (DynamicDockerContainer.class.isAssignableFrom(fixture)) {
```

Comment:
Works as expected - but the portmapping was still getting in the way for the FTP PASSV connections.